exercise-1/index.html

```diff
@@ -6,18 +6,28 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Exercise 1</title>
     <link rel="stylesheet" href="./style.css" />
+ <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
I can see a number of issues here with indentation, unclosed self closing tags (e.g. `<link>` vs `<link />`). I recommend you install [the Prettier code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) and set it as your default formatter for VSCode, it will save you a lot of manual work and it will make your code look cleaner and easier to read.